src/azure-cli-core/azure/cli/core/profiles/_shared.py

```diff
@@ -152,7 +152,7 @@ def default_api_version(self):
         ResourceType.MGMT_RESOURCE_DEPLOYMENTSCRIPTS: '2019-10-01-preview',
         ResourceType.MGMT_NETWORK_DNS: '2018-05-01',
         ResourceType.MGMT_KEYVAULT: '2019-09-01',
-        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2018-09-01-preview', {
+        ResourceType.MGMT_AUTHORIZATION: SDKProfile('2020-04-01-preview', {
             'classic_administrators': '2015-06-01',
             'role_definitions': '2018-01-01-preview',
```

Comment:
Do we need update this line?

Reply:
`2018-01-01-preview` is the latest version which contains `RoleDefinition`.